dotlottie-rs/src/dotlottie_player.rs

```diff
@@ -1141,6 +1184,598 @@ impl DotLottiePlayer {
+                let values: Vec<&str> = value.split_whitespace().collect();
+                if values.len() != 2 {
+                    return false;
+                }
+                let pointer_event = Event::OnPointerMove {
+                    x: values[0].parse::<f32>().map_err(|_| ()).unwrap(),
+                    y: values[1].parse::<f32>().map_err(|_| ()).unwrap(),
+                };
+                match self.state_machine.try_write() {
+                    Ok(mut state_machine) => {
+                        if let Some(sm) = state_machine.as_mut() {
+                            sm.post_event(&pointer_event);
+                        }
+                    }
+                    Err(_) => return false,
+                }
+            }
+            "OnPointerEnter" => {
+                let values: Vec<&str> = value.split_whitespace().collect();
+                if values.len() != 2 {
+                    return false;
+                }
+                let pointer_event = Event::OnPointerEnter {
+                    x: values[0].parse::<f32>().map_err(|_| ()).unwrap(),
+                    y: values[1].parse::<f32>().map_err(|_| ()).unwrap(),
+                };
+                match self.state_machine.try_write() {
+                    Ok(mut state_machine) => {
+                        if let Some(sm) = state_machine.as_mut() {
+                            sm.post_event(&pointer_event);
+                        }
+                    }
+                    Err(_) => return false,
+                }
+            }
+            "OnPointerExit" => {
+                let values: Vec<&str> = value.split_whitespace().collect();
+                if values.len() != 2 {
+                    return false;
+                }
+                let pointer_event = Event::OnPointerExit {};
+                match self.state_machine.try_write() {
+                    Ok(mut state_machine) => {
+                        if let Some(sm) = state_machine.as_mut() {
+                            sm.post_event(&pointer_event);
+                        }
+                    }
+                    Err(_) => return false,
+                }
+            }
+            "OnComplete" => {
+                let pointer_event = Event::OnComplete {};
+
+                match self.state_machine.try_write() {
+                    Ok(mut state_machine) => {
+                        if let Some(sm) = state_machine.as_mut() {
+                            sm.post_event(&pointer_event);
+                        }
+                    }
+                    Err(_) => return false,
+                }
+            }
+            _ => return false,
+        }
+        return true;
+    }
+
+    pub fn load_animation_path(&self, animation_path: &str, width: u32, height: u32) -> bool {
+        let is_ok = self
+            .player
+            .write()
+            .is_ok_and(|runtime| runtime.load_animation_path(animation_path, width, height));
+
+        is_ok
```

Comment:
we can return directly, no need to assign in is_ok var 
```suggestion
       self
            .player
            .write()
            .is_ok_and(|runtime| runtime.load_animation_path(animation_path, width, height))
```